pkg/operator/ceph/cluster/nodedaemon/reconcile.go

```diff
@@ -85,6 +85,18 @@ func (r *ReconcileNode) Reconcile(context context.Context, request reconcile.Req
 	return result, err
 }
 
+func (r *ReconcileNode) cleanupExporterResources(clusterNamespace string, ns string, nodeName string) (reconcile.Result, error) {
```

Comment:
no need to return an error, to fix the linter issue
```suggestion
func (r *ReconcileNode) cleanupExporterResources(clusterNamespace string, ns string, nodeName string) reconcile.Result {
```

Reply:
Actually I take that back, the error is still needed, so we will need to return the error from the caller, like the linter suggests